app/src/main/res/layout-sw600dp-port/option_story_text_size.xml

```diff
@@ -20,9 +20,9 @@
     android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"
```

Comment:
This file should not be changed. 
1. Don't change anything in this file.
2. Shift this file back to `layout-sw600dp` folder.

Reply:
@rt4914 If I revert changes here I will have to bring back the land file for the tablet too

Reply:
Why do you need to make changes for land file? 
Just for understanding
`sw600dp-land` -> tablet landscape file
`sw600dp-port` -> tablet portrait file
`sw600dp` -> tablet landscape + portrait file


And this PR is making a change such that the `sw600dp` is shifted to `sw600dp-port` which means we won't have landscape file for tablet.